Add `internal/params` pacakge test (#474)
vdaas/vald: docs/contributing/coding-style.md

}
```

### Unused Variables

An unused variable may increase the complexity of the source code, it may confuse the developer hence introduce a new bug.
So please delete the unused variable.

> Generally, the unused variable should be reported during compilation, but in some cases, the compiler may not report an error. This is an example of the unused variable declaration that does not cause a compilation error.<br />

Reviewer: ```suggestion
Generally, the unused variable should be reported during compilation, but in some cases, the compiler may not report an error. This is an example of the unused variable declaration that does not cause a compilation error.
```

> ```go

Reviewer: ```suggestion
```golang
```
we should unify using word

type server struct {
    addr string
    port int
}

srv := &server {
    addr: "192.168.33.10:1234",

>     // port <- unused variables

Reviewer: I'm not sure this line 😕 
Could you please describe about this line?

Author: This variable represents unused variable.
If you do not use this field,  you can delete the port definition in the field struct. In other words, it means to erase the unused value.

Reviewer: Do you mean the variable which is commented out will be ignored Go compilation?
The description what I want to is what cause in this situation.

}

if err := srv.Run(); err != nil {
    log.Fatal(err)
}

> 

Reviewer: ```suggestion
```

}
```

### Unused Variables

> It is an error to declare a variable without using it.

Reviewer 2: I think the section `unused Variables` from the `test` section is better, can you move back to here?

> while a variable that is initialized but not used is at least a wasted computation and perhaps indicative of a larger bug.

Reviewer 2: ```suggestion
Unused variable may increase the complexity of the source code, it may confuse the developer hence introduce a new bug.
```

> So please delete unused variables.

Reviewer 2: ```suggestion
So please delete the unused variable.
```

> The following is an example of an unused variables that does not cause an error.<br>

Reviewer 2: ```suggestion
Generally the unused variable should be reported during compilation, but in some case the compiler may not report an error. This is an example of the unused variable declaration that does not cause an compilation error.<br>
```

> Please delete any variables that you don't need, as they can create bugs.

Reviewer 2: ```suggestion
```

> Generally, the unused variable should be reported during compilation, but in some cases, the compiler may not report an error. This is an example of the unused variable declaration that does not cause a compilation error.

Reviewer: ```suggestion
Generally, the unused variable should be reported during compilation, but in some cases, the compiler may not report an error. 
This is an example of the unused variable declaration that does not cause a compilation error.
```

// In this case, this example are not using `port` field, but dose not cause a compilation error.
// So please delete `port` field of `server`.

type server struct {
    addr string

>     port int

Reviewer: ```suggestion
    port int  // you have to delete this field.
```

}

> // The port number is included in `addr`, so the `port` field of `server` is not used.

Reviewer: ```suggestion
// The `port` field of `server` is not used.
```

        }
        // generated test code
    ```

1. Unused fields

    By default, the template provides `fields` structure to initialize object of the test target. But in some cases, not all `fields` are needed. So please delete unnecessary fields.

    If the test case needs only `addr` of `fields`, please delete other fields.

> 

Reviewer 2: I think it is better to provide the example of the struct and the corresponding test code.

        }
        // generated test code
    ```

1. Unused fields

    By default, the template provides `fields` structure to initialize object of the test target. 
    
    But in some cases, not all `fields` are needed, so please delete the unnecessary fields.

>     If the test case needs only `addr` of `fields`, please delete other fields.

Reviewer 2: ```suggestion
    For example the following struct and the corresponding function:
    
    ```golang
    type server struct {
        addr string
        port int
    }
    func (s *server) Addr() string {
        return s.addr
     }
    ```

    And the generated test code is:
    ```golang
    func Test_server_Addr(t *testing.T) {
        type fields struct {
            addr string
            port int
	}
	type want struct {
        // generated test code
    ```

    If the test case needs only `addr` of `fields`, please delete the unused field like `port`.

    ```golang
    func Test_server_Addr(t *testing.T) {
        type fields struct {
            addr string
            // port int   <-- this line should be deleted
	}
	type want struct {
        // generated test code
    ```

```

>     ```go

Reviewer 2: if you accept my suggestion, this block should be deleted :)

        }
        // generated test code
    ```

1. Unused fields

    By default, the template provides `fields` structure to initialize object of the test target. 

>     

Reviewer: ```suggestion
```

    But in some cases, not all `fields` are needed, so please delete the unnecessary fields.

> 

Reviewer: ```suggestion
```

    For example, the following struct and the corresponding function:
    
    ```golang
    type server struct {
        addr string
        port int
    }
    func (s *server) Addr() string {
        return s.addr

>      }

Reviewer 2: ```suggestion
    }
```

    ```

    And the generated test code is:
    ```golang
    func Test_server_Addr(t *testing.T) {
        type fields struct {
            addr string
            port int

> 	}

Reviewer 2: ```suggestion
    }
```

>     }

Reviewer 2: ```suggestion
        }
```

> 	type want struct {

Reviewer 2: ```suggestion
    type want struct {
```

>     type want struct {

Reviewer 2: ```suggestion
        type want struct {
```

>         // generated test code

Reviewer 2: ```suggestion
    // generated test code
```

>     // generated test code

Reviewer 2: ```suggestion
        // generated test code
```

    ```

    Since the `port` variable is not used in this test case, you can delete the `port` definition in the test case.

> 

Reviewer: ```suggestion
```

    ```golang
    func Test_server_Addr(t *testing.T) {
        type fields struct {
            addr string
            // port int   <-- this line should be deleted

> 	}

Reviewer 2: ```suggestion
    }
```

> 	}

Reviewer 2: ```suggestion
        }
```

> 	type want struct {

Reviewer 2: ```suggestion
    type want struct {
```

>     type want struct {

Reviewer 2: ```suggestion
        type want struct {
```

>         // generated test code

Reviewer 2: ```suggestion
    // generated test code
```

>     // generated test code

Reviewer 2: ```suggestion
        // generated test code
```

>     ```

Reviewer 2: ```suggestion
    ```
```

        }
        // generated test code
    ```

> 1. Unused Variables

Reviewer 2: sorry for misunderstandable comment before but I meant in this part we should write to delete the unused fields here

        }
        // generated test code
    ```

1. Unused Variables

>     Unused variable may increase the complexity of the source code, it may confuse the developer hence introduce a new bug.

Reviewer 2: ```suggestion
    An unused variable may increase the complexity of the source code, it may confuse the developer hence introduce a new bug.
```

    So please delete the unused variable.
    

>     Generally the unused variable should be reported during compilation, but in some case the compiler may not report an error. This is an example of the unused variable declaration that does not cause an compilation error.<br>

Reviewer 2: ```suggestion
Generally, the unused variable should be reported during compilation, but in some cases, the compiler may not report an error. This is an example of the unused variable declaration that does not cause a compilation error.<br />
```
